src/test/kotlin/com/terraformation/backend/db/DatabaseTest.kt

```diff
@@ -2242,6 +2270,7 @@ abstract class DatabaseTest {
     val observationIds = mutableListOf<ObservationId>()
     val organizationIds = mutableListOf<OrganizationId>()
     val participantIds = mutableListOf<ParticipantId>()
+    val participantProjectSpeciesIds = mutableListOf<ParticipantProjectSpeciesId>()
```

Comment:
Maybe also add `participantProjectSpeciesId` to return the most recently-inserted ID?